oshmem/mca/sshmem/ucx/sshmem_ucx_module.c

```diff
@@ -104,7 +97,7 @@ static segment_allocator_t sshmem_ucx_allocator = {
 
 static int
 segment_create_internal(map_segment_t *ds_buf, void *address, size_t size,
-                        unsigned flags, long hint, void *dev_mem)
+                        unsigned flags, long hint)
```

Comment:
pass memory type instead of hint.
this will align with how "flags" parameter is used